src/listings/reserved_ranges.md

```diff
@@ -0,0 +1,94 @@
+|---|---|
+| Method_StorageMinerActor_OnDeferredCronEvent        | # |
+|	Method_StorageMinerActor_PreCommitSector            | # |
+|	Method_StorageMinerActor_ProveCommitSector          | # |
+|	Method_StorageMinerActor_DeclareTemporaryFaults     | # |
+|	Method_StorageMinerActor_RecoverTemporaryFaults     | # |
+|	Method_StorageMinerActor_ExtendSectorExpiration     | # |
+|	Method_StorageMinerActor_TerminateSector            | # |
+|	Method_StorageMinerActor_SubmitSurprisePoStResponse | # |
+|	Method_StorageMinerActor_OnVerifiedElectionPoSt     | # |
+|	Method_StorageMinerActor_OnSurprisePoStChallenge    | # |
+|	Method_StorageMinerActor_GetPoStState               | # |
+|	Method_StorageMinerActor_GetOwnerAddr               | # |
+|	Method_StorageMinerActor_GetWorkerAddr              | # |
+|	Method_StorageMinerActor_GetWorkerVRFKey            | # |
+
+## StorageMarketActor Methods
+| Method                                                                  | ID |
+|---|---|
+| Method_StorageMarketActor_OnEpochTickEnd                                | 1 |
+| Method_StorageMarketActor_AddBalance                                    | 2 |
+|	Method_StorageMarketActor_WithdrawBalance                               | 3 |
+|	Method_StorageMarketActor_PublishStorageDeals                           | 4 |
+|	Method_StorageMarketActor_OnMinerSectorPreCommit_VerifyDealsOrAbort     | 5 |
+|	Method_StorageMarketActor_OnMinerSectorProveCommit_VerifyDealsOrAbort   | 6 |
+|	Method_StorageMarketActor_OnMinerSectorsTerminate                       | 7 |
+|	Method_StorageMarketActor_GetPieceInfosForDealIDs                       | 8 |
+|	Method_StorageMarketActor_GetWeightForDealSet                           | 9 |
+
+## StoragePowerActor Methods
+| Method                                                        | ID |
+|---|---|
+|	Method_StoragePowerActor_OnEpochTickEnd                       | 1 |
+|	Method_StoragePowerActor_AddBalance                           | 2 |
+|	Method_StoragePowerActor_WithdrawBalance                      | 3 |
+|	Method_StoragePowerActor_CreateMiner                          | 4 |
+|	Method_StoragePowerActor_DeleteMiner                          | 5 |
+|	Method_StoragePowerActor_ReportConsensusFault                 | 6 |
+|	Method_StoragePowerActor_OnSectorProveCommit                  | 7 |
+|	Method_StoragePowerActor_OnSectorTemporaryFaultEffectiveBegin | 8 |
+|	Method_StoragePowerActor_OnSectorTemporaryFaultEffectiveEnd   | 9 |
+|	Method_StoragePowerActor_OnSectorModifyWeightDesc             | 10 |
+|	Method_StoragePowerActor_OnSectorTerminate                    | 11 |
+|	Method_StoragePowerActor_OnMinerSurprisePoStSuccess           | 12 |
+|	Method_StoragePowerActor_OnMinerSurprisePoStFailure           | 13 |
+|	Method_StoragePowerActor_OnMinerEnrollCronEvent               | 14 |
+|	Method_StoragePowerActor_GetMinerConsensusPower               | 15 |
+|	Method_StoragePowerActor_GetMinerUnmetPledgeCollateralRequirement | 16 |
```

Comment:
I'm not sure it's a good idea to repeat these here. They need to be fully specified by the actors (which will move out of this repo), but there will probably be a bit of churn and these two representations will depart.

As far as reserved ranges go, we reserve method ID `0` for value-send and `1` for constructor, and all other positive values are free for actors to use.

Reply:
I removed the full methods listing, and added a TODO (our favourite) to link to actor code when done.